xml/dm_registers.xml

```diff
@@ -153,8 +148,8 @@
             Debug Module after power up, including the platform's system reset
             or Debug Transport reset signals.
 
-            A debugger should pulse this bit low to ensure that the Debug
-            Module is fully reset and ready to use.
+            When first connecting, a debugger should pulse this bit low to
```

Comment:
I would like for it to be in the spec, so that compliant debuggers will perform the reset when connecting, and will work with targets that need that reset. (I can't think of any unhappy side effects pulsing dmactive.)